src/darwin/CHIPTool/CHIPTool.xcodeproj/project.pbxproj

```diff
@@ -558,15 +558,15 @@
 				CODE_SIGN_ENTITLEMENTS = CHIPTool/CHIPTool.entitlements;
 				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = W4W8M6J397;
```

Comment:
This and other changes in this file are presumably not intended?

Reply:
not intended